Composer/packages/client/src/pages/design/DebugPanel/DebugPanel.tsx

```diff
@@ -3,12 +3,14 @@
 
 /** @jsx jsx */
 import { jsx, css } from '@emotion/core';
-import { useMemo, useCallback, useRef, useEffect } from 'react';
+import React, { useMemo, useCallback, useRef, useEffect } from 'react';
```

Comment:
are the changes below relevant to this PR? @srinaath

Reply:
So this was a piece uncaught from the previous PR. Its related to the webchat Inspector and makes sense to funnel in this PR.

Reply:
React was also generating a warning as this component went agains HTML semantics by having a button inside a button